github/ci/prow-deploy/tasks/deploy.yml

```diff
@@ -34,6 +34,25 @@
     KUBECONFIG: "{{ kubeconfig_path }}"
   when: bootstrap_full_config
 
+- name: Reconcile webhooks for onboarded repositories
+  shell: |
+    hmac \
+      --config-path=$PROW_CONFIG_PATH \
+      --github-token-path=$GITHUB_TOKEN_PATH \
+      --kubeconfig=$KUBECONFIG \
+      --kubeconfig-context=ibm-prow-jobs \
+      --hmac-token-secret-namespace={{ prowNamespace }} \
+      --hmac-token-secret-name=hmac-token \
+      --hmac-token-key=hmac \
+      --hook-url https://prow.ci.kubevirt.io/hook \
+      --deck-url https://prow.ci.kubevirt.io/ \
+      --dry-run=true
```

Comment:
This is likely going to fail, tried it locally, maybe just set to false after we find a way to test this in the test deploy?